cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -366,6 +367,7 @@ def aqt_compile(
                 implies css.CZ3 for all "CZ3/*" calibrations except "CZ3/C5T4", which will be
                 mapped to a css.CZ3_INV on qutrits (4, 5). Setting any calibration to None will
                 disable that calibration.
+            options: dictionary of desired aqt_compile options.
```

Comment:
nit: argument descriptions should be capitalized (though admittedly this isn't consistent across the repo yet)
```suggestion
            options: Dictionary of desired aqt_compile options.
```
(ditto elsewhere)

Reply:
Using `kwargs` means code that calls this function would not need to be updated when more options are added to the function signature, which seems like a plus. 

The only downside I can think of is that it might be confusing, but that can be kinda fixed through writing what `kwargs` does in the docstring.  

I like that suggestion, I'll update the code to use that unless there are any objections.